src/state/workload.rs

```diff
@@ -0,0 +1,1101 @@
+// Copyright Istio Authors
```

Comment:
Note to reviewers: This is just a move of `workload.rs`, minus the `WorkloadInformation` and `WorkloadManager`.